lib/amazon-pricing/definitions/database-type.rb

```diff
@@ -105,7 +104,7 @@ def self.display_name(name)
   	def self.get_database_name
       [:mysql, :postgresql, :oracle_se1, :oracle_se, :oracle_ee, :sqlserver_ex, :sqlserver_web,
         :sqlserver_se, :sqlserver_ee, :aurora, :mariadb,
-        :oracle_se2 # oracle_se2 prices are copied, not collected
+        :oracle_se2 # oracle_se2 now collected (but not byol)
```

Comment:
let's make this comment a little better (took me time to understand):

oracle_se2 license included prices are collected, and BYOL prices are copied